include/cuda/std/detail/libcxx/include/__mdspan/standard_layout_static_array.hpp

```diff
@@ -534,17 +534,9 @@ struct __standard_layout_psa<_Tag, _Tp, _static_t, _CUDA_VSTD::integer_sequence<
   //--------------------------------------------------------------------------
 
   __MDSPAN_INLINE_FUNCTION_DEFAULTED
-  constexpr __standard_layout_psa() noexcept
-#if defined(__clang__) || defined(__MDSPAN_DEFAULTED_CONSTRUCTORS_INHERITANCE_WORKAROUND)
```

Comment:
@youyu3 
Did the above change fix this issue, making this Clang work-around no longer needed?

Reply:
In my tests with clang, yes the work-around is no longer needed.